tensorboard/webapp/experiments/types.ts

```diff
@@ -20,4 +20,5 @@ export interface Experiment {
   description?: string;
   hparams?: string;
   tags?: string[];
+  related_links?: Array<{name: string; url: string}>;
```

Comment:
nit: while start_time is snake case already, can we use camelCase?

Reply:
I indeed used snake case here to be consistent with `start_time`, since
these responses come directly from the backend (which uses snake case).

Since we don’t have a backend-to-frontend translation, either camel case
or snake case will be inconsistent with the conventions on one end.
Given that, I’d like to keep this consistent with the existing fields,
so that it’s not inconsistent with _both_ ends! If this bothers you, the
right solution imho is to teach either the backend or frontend to
automatically translate from one form to the other.

Reply:
![image](https://user-images.githubusercontent.com/710113/97622549-a7e3eb00-19e1-11eb-9e5d-ec0edbdacfa4.png)

Reply:
kebab-case for Racket!

Reply:
I need to start a trend to call names joined with periods `domain.case` - though really it'd be better as `case.domain` to do it right-to-left just to confuse everyone the same way DNS does.